examples/example-userspace/examples/tc-map-share.rs

```diff
@@ -27,7 +27,7 @@ async fn main() {
         .with_max_level(Level::TRACE)
         .finish();
     tracing::subscriber::set_global_default(subscriber).unwrap();
-    if unsafe { libc::getuid() != 0 } {
+    if unsafe { libc::geteuid() != 0 } {
```

Comment:
Just to note that the `geteuid()` changes are not strictly related to this patchset.

Reply:
That's true. It would be better this modification is sent via separate PR.